app/src/org/commcare/utils/PhoneNumberHelper.java

```diff
@@ -134,9 +132,12 @@ public static void requestPhoneNumberHint(ActivityResultLauncher<IntentSenderReq
                         IntentSenderRequest intentSenderRequest = new IntentSenderRequest.Builder(pendingIntent).build();
                         phoneNumberHintLauncher.launch(intentSenderRequest);
                     } catch (Exception e) {
+                        FirebaseAnalyticsUtil.reportPhoneNumberHintFailure(e.getMessage());
```

Comment:
we don't want to log this to analytics but only with `Logger.exception` as non-fatals